swan-lake/learn/documenting-ballerina-code.md

```diff
@@ -95,56 +95,68 @@ public function submit(@sensitive string httpVerb, string path, Request request)
 
 A Ballerina module can have a `Module.md` file, which describes the module and its usage.
 
-A typical project structure of a Ballerina project is like this:
+A typical package structure of a Ballerina package is like this:
 
 ```
 /
-  Ballerina.toml       # Configuration that defines project intent
-    src
-      module1/             # The source in this directory will be named “<org-name>/module1”
+  Ballerina.toml       # Configuration that defines package intent
+  main.bal
+  Module.md            # Optional, contains descriptive metadata of the default module for display at
+                       # Ballerina Central        
+  Package.md           # Optional, contains descriptive metadata of the package for display at
+                       # Ballerina Central  
+    modules
+      module1/             
         Module.md          # Optional, contains descriptive metadata for display at
                            # Ballerina Central
         *.bal
         [tests/]           # Module-specific unit and integration tests
         [resources/]       # Module-specific resources
 
-      modules.can.include.dots.in.dir.name/
+      module2/
         Module.md
         *.bal
         [tests/]
         [resources/]
 
-  [resources/]         # Resources included with every module in the project
+  [resources/]         # Resources included with every module in the package
 
   target/              # Compiled executables and other artifacts end up here
 ```
 
-The `ballerina doc` command will read the `Module.md` and prepend it to the generated HTML file.
+The `ballerina doc` command will read the `Package.md` and `Module.md` and prepend it to the generated HTML file.
 
 Check [HTTP module documentation](/swan-lake/learn/api-docs/ballerina/http/index.html) for sample HTML that has `Module.md` content at the top, followed by the other module constructs.
 
 
 ## Generating Ballerina Documentation
 
-Ballerina provides a `doc` command, which can be executed against a given Ballerina project. This command will result in generating the Ballerina documentation as HTML files, for all the modules in the project.
+Ballerina provides a `doc` command, which can be executed against a given Ballerina package. This command will result in generating the Ballerina documentation as HTML files, for all the modules in the package.
 
-First, let's create a new Ballerina project:
+First, let's create a new Ballerina package:
 
 ```bash
-$ ballerina new myproject
-Created new Ballerina project at myproject
+$ ballerina new math
+Created new Ballerina package 'math' at math
```

Comment:
```suggestion
Created new Ballerina package 'math' at math
```
What is the second "math" here?

Reply:
This is the console output from ballerina new command

Reply:
Ack. Fine.